bootstrap/cmd/kfctl/cmd/root.go

```diff
@@ -131,23 +137,26 @@ func loadKfApp(options map[string]interface{}) (kftypes.KfApp, error) {
 	if kAppErr != nil {
 		return nil, fmt.Errorf("there was a problem loading app %v. Error: %v", appName, kAppErr)
 	}
-	ksApp := &kstypes.Ksonnet{}
-	dat, datErr := ioutil.ReadFile(cfgfile)
-	if datErr != nil {
-		return nil, fmt.Errorf("couldn't read %v. Error: %v", cfgfile, datErr)
+	buf, bufErr := ioutil.ReadFile(cfgfile)
```

Comment:
this line effectively requires us to run kfctl within appDir, which will cause problems later on when running `kfctl generate all`